sormas-ui/src/main/java/de/symeda/sormas/ui/person/PersonsView.java

```diff
@@ -181,15 +203,16 @@ private void updateAssociationButtons() {
 
 		associationButtons.keySet().forEach(b -> {
 			CssStyles.style(b, CssStyles.BUTTON_FILTER_LIGHT);
-			b.setCaption(associationButtons.get(b));
+			b.setCaption(PersonAssociation.valueOf(associationButtons.get(b)).toString());
 			if (b.getData() == criteria.getPersonAssociation()) {
 				activeAssociationButton = b;
 			}
 		});
 		if (activeAssociationButton != null) {
 			CssStyles.removeStyles(activeAssociationButton, CssStyles.BUTTON_FILTER_LIGHT);
 			activeAssociationButton.setCaption(
-				associationButtons.get(activeAssociationButton) + LayoutUtil.spanCss(CssStyles.BADGE, String.valueOf(grid.getItemCount())));
+				PersonAssociation.valueOf(associationButtons.get(activeAssociationButton))
```

Comment:
I don't get why this was changed, the map contains captions as values (for whatever reason)